lib/fs.js

```diff
@@ -1842,7 +1842,7 @@ ReadStream.prototype.close = function(cb) {
       this.once('open', close);
       return;
     }
-    return process.nextTick(this.emit.bind(this, 'close'));
```

Comment:
Also this case is particularly easy -- no other arguments will need to be passed because none were passed to `nextTick`, which doesn't pass any on it's own.